sealer/cmd/build.go

```diff
@@ -65,6 +68,7 @@ func init() {
 	buildCmd.Flags().StringVarP(&buildConfig.BuildType, "buildType", "b", "", "cluster image build type,default is cloud")
 	buildCmd.Flags().StringVarP(&buildConfig.KubefileName, "kubefile", "f", "Kubefile", "kubefile filepath")
 	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageName", "t", "", "cluster image name")
+	buildCmd.Flags().StringVarP(&buildConfig.ImageName, "imageList", "i", "", "build by submitting image list")
```

Comment:
Just using $rootfs/manifests/imageList file, for most users don't care about this argument.

Reply:
OK